README.md

```diff
@@ -6,27 +6,27 @@
 
 The Data Processing Framework (DPF) is designed to provide numerical
 simulation users/engineers with a toolbox for accessing and
-transforming simulation data. DPF can access data from solver result
-files as well as several neutral formats (csv, hdf5, vtk,
-etc.). Various operators are available allowing the manipulation and
-the transformation of this data.
+transforming simulation data.
 
-The Python `ansys-dpf-post` package provides a simplified Python
-interface to DPF, thus enabling rapid postprocessing without ever
-leaving a Python environment. 
+The Python `ansys-dpf-post` package provides a high level, physics oriented API for postprocessing.
+Loading a simulation (defined by its result files) allows to extract simulation metadata as well
```

Comment:
Either "allows you to extract simulation...." or "(defined by its result files) extracts simulation metadata....."

Reply:
Thanks @JennaPaikowsky I'll go with the first option as it's not loading the simulation that extracts but it's something one can after loading the simulation
```suggestion
Loading a simulation (defined by its result files) allows you to extract simulation metadata as well
```